h2o-hadoop-common/tests/python/pyunit_gcs_import.py

```diff
@@ -1,18 +0,0 @@
-#! /usr/env/python
-
-import sys
-import os
-sys.path.insert(1, os.path.join("../../../h2o-py"))
-from tests import pyunit_utils
-import h2o
-
-def gcs_import():
-    # Just test the import works - no class clashes, no exception
```

Comment:
I think we should keep running this test on hdp, why not?

Reply:
Oh my god, I did not intend to move it.

Reply:
Moved back.